forem/forem
# 17222  remove spaminess rating from articles

## What type of PR is this? (check all applicable)


## Description
This needs to be deployed after [this PR](https://github.com/forem/forem/pull/17846) is merged.

We've removed references to article `spaminess_rating` from the codebase. Then we [ignored the column on the model](https://github.com/forem/forem/pull/17846), so we can safely remove it from the database.

This PR contains the migration to remove the `spaminess_rating` column from the `articles` table.

## Related Tickets & Documents
- Closes #17222 

## QA Instructions, Screenshots, Recordings
- run the migration with `rails db:migrate`
- go into the rails console `rails c`
- try to query the spaminess rating of an article: `Article.last.spaminess_rating`
- you should get the following error: `NoMethodError: undefined method `spaminess_rating' for #<Article ...`

### UI accessibility concerns?
N/A

## Added/updated tests?

- [x] No, and this is why: all tests related to an article's `spaminess_rating` have already been removed

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
